rails-commenting.rb

```diff
@@ -36,15 +37,15 @@ def destroy
     if @post.delete
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6)deletes post and redirects it to post
       redirect_to blog_post_path(@post)
     end
   end
 
   # 7)
   private
   def blog_post_params
-    # 8)
+    # 8)must be added to or will receive error
```

Comment:
setting up strong params, help with security and only get the information we want in the database